examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<ul><li>"leave it for later ..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Fixed.